configapi/api_default.go

```diff
@@ -174,7 +180,7 @@ func GetNetworkSliceByName(c *gin.Context) {
 	if errGetOne != nil {
 		logger.DbLog.Warnln(errGetOne)
 	}
-	json.Unmarshal(mapToByte(rawNetworkSlice), &networkSlice)
+	json.Unmarshal(configmodels.MapToByte(rawNetworkSlice), &networkSlice)
```

Comment:
Error is not handled here.

Reply:
This modification is out of the scope of this PR